saskatoon/harvest/filters.py

```diff
@@ -191,6 +195,16 @@ class Meta:
     )
 
 
+class EquipmentPointFilter(OrganizationFilter):
```

Comment:
subclassing `OrganizationFilter` means it will get the `is_equipment_point` field, which we don't need. Please create a new, independant filter by redefining the `Meta class`